app/pages/Accounts/Accounts.module.scss

```
@@ -90,3 +90,15 @@
     top: calc(50% - 17px);
     height: 34px;
 }
+
+.displayAddressQRBig {
+    height: calc(min(512px, 70%));
+    width: calc(min(512px, 70%));
+    margin-bottom: 50px;
```

To align on small views:

```suggestion
    margin-top: 50px;
    margin-bottom: 50px;
```